ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -388,42 +388,32 @@ objects:
           echo "Gathering artifacts ..."
           mkdir -p /tmp/artifacts/pods /tmp/artifacts/nodes /tmp/artifacts/metrics /tmp/artifacts/bootstrap /tmp/artifacts/network
 
-
-          if [ -f /tmp/artifacts/installer/terraform.tfstate ]
+          export PREFIX=$(grep cluster_id /tmp/artifacts/installer/terraform.tfvars | cut -d '"' -f4)
+          bootstrap_ip=$(openstack floating ip list --long -f csv -c Description -c "Floating IP Address" | grep "$PREFIX-bootstrap-fip" | cut -f1 -d "," | sed -e "s|\"||g")
```

Comment:
Isn't there a risk it gets the FIP of other jobs if PREFIX is empty?

Reply:
I'm obtaining the PREFIX here the same way we do it in the 'nuke' function above (line 354). So if it fails here I suppose we'd also be leaking CI environments in general. We could refactor things to set PREFIX once I guess

Reply:
Also, the ssh keys in each environment should be different so even if PREFIX somehow got set incorrectly the logs should fail to copy over.